docs/userguide/envvars.md

```diff
@@ -74,6 +74,18 @@ are being shipped in the default dependencies because there is no good way of cr
 opt-out dependencies, but the baybe package will work without them.
 ```
 
+## Polars
+If baybe was installed with the additional Polars dependency (`baybe[polars]`), it will
```

Comment:
baybe vs. `baybe` vs. BayBE vs. `BayBE`, how do we write this in other places?

Reply:
this clearly speaks about the python package, hence I went with `baybe`, is that not right?

Reply:
I think we are always talking about the package in whatever context here in the repo :D
So I think I agree with @AVHopp, should be `BayBE` – that's also how we do it everywhere else. Except these two cases, which should perhaps be changed, too
<img width="390" alt="image" src="https://github.com/user-attachments/assets/4f988b7c-b5e9-472f-84ec-daab5b71b219">
In the first case, one could simply say `but BayBE will work ...` and in the second it should also be `BayBE`

Reply:
ok changed